### PR TITLE
Add requirements.txt

### PR DESCRIPTION
**Pull request by rodrigoborgesdeoliveira**
Add `requirements.txt` for easier dependencies installation.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+tweepy
+spaCy
+folium
+geopy
+tqdm
+geograpy3
```
Comment (pawelad) [inline requirements.txt:6]: You could pin requirements to the latest compatible release (`~=`)  to make the environment more reproducible.

i.e. `tweepy~=4.6.0` instead `tweepy`.

**Review by pawelad — COMMENTED** [review]
Nice addition 👍

**Comment by kinshukdua** [comment]
Thanks for the PR @rodrigoborgesdeoliveira!

@pawelad I guess we can add specific versions later on, right now the main focus is on improving functionality. But feel free to create a PR and I'll merge it.

**Comment by pawelad** [comment]
No worries, I just wanted to share some good practices : )

**Pull request merged**